docs/checks/internal/checks.yaml

```diff
@@ -18,6 +18,7 @@ checks:
   Maintained:
     risk: High
     tags: supply-chain, security
+    repos: GitHub
```

Comment:
Maybe simpler to add a boolean like `run-on-local-files`? That way `checks.yaml` doesn't have to be updated with newer repo implementations.

Reply:
If you don't mind, I'll keep it as-is because it lends itself well to supporting other interfaces: gitlab, local-git, etc. I don't think we'll be able to support all of them with a binary "run on local file or not".